pages/etherpad/[[...roomId]].js

```diff
@@ -213,10 +211,11 @@ export default function Etherpad() {
                         { getConfig().publicRuntimeConfig.authProviders.etherpad.myPads?.api && !serverPads && <ErrorMessage>{ t('Can\'t connect to the provided /write server. Please try again later.') }</ErrorMessage> }
                         <ServiceTable>
                             { matrix.spaces.get(matrix.serviceSpaces.etherpad).children?.map(writeRoomId => {
-                                return <EtherpadListEntry
+                                return <ServiceLink
                                     key={writeRoomId}
                                     roomId={writeRoomId}
-                                    padName={_.get(matrix.rooms.get(writeRoomId), 'name')}
+                                    name={_.get(matrix.rooms.get(writeRoomId), 'name')}
+                                    path={getConfig().publicRuntimeConfig.authProviders.etherpad.path?.replace(/[<>\s/:]/g, '') || 'etherpad'}  // sanitizing the string just in case of any forbidden url characters
```

Comment:
This shouldn't be necessary... one should sanitize user input, yes. But sanitizing a value that was put in by a sysadmin/developer/engineer shouldn't be necessary. Also, let's take care of this in a separate PR please. It's basically the issue that I opened up here, that you're starting to address: https://github.com/medienhaus/medienhaus-spaces/issues/30

And if I understood the NextJS documentation correctly, it should be totally fine to just put `etherpad` here. The router will redirect you to, let's say `write` for example, if that's the path alias that was set up in the config file.